### ISSUE TITLE
Inpaint only the highest confidence detection

### ISSUE THREAD

**Issue by GreenLandisaLie** [Bing-su/adetailer#340]
### Question

Better documentation regarding the existing options would be helpful and a tooltip of sorts when hovering the mouse over them would be extra useful.

I was under the impression the 'top K' feature would make it so Adetailer would sort each detection by condidence (desc) and apply inpainting to each detection on that list up to the number defined by that setting. But since there is no documentation on it - I guess I was wrong since that did not happen. If it is then I guess its bugged?

I got 2 detections using a nsfw model (1 - 50%~, 2 - >70%) and even though I set the 'top K' feature to '1' it still inpainted both detections starting with the lowest confidence one. **v23.8.1**

**TLDR:** What does the 'top K' option does and is there a way to limit the number of inpaints by X given amount regardless of the number of detections found (and starting with the one that has the highest confidence)?
